clients/admin-ui/src/features/configure-consent/AddVendor.tsx

```diff
@@ -139,15 +183,15 @@ const AddVendor = () => {
             <AddModal
               isOpen={modal.isOpen}
               onClose={modal.onClose}
-              title="Add a vendor"
+              title={passedInSystem ? "Edit vendor" : "Add a vendor"}
               onSuggestionClick={() => {
                 setIsShowingSuggestions(true);
```

Comment:
I suspect we'll want something more defensive for showing suggestions when it's a passed in system. something like what we have in the data uses form where it says current ones would be overwritten? might want to ask Rachel what she thinks should happen